staging/src/k8s.io/kube-apiextensions-server/pkg/apiserver/customresource_handler.go

```diff
@@ -174,14 +176,26 @@ func (r *crdHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 		handler(w, req)
 		return
 	case "create":
+		if terminating {
+			http.Error(w, fmt.Sprintf("%v not allowed while CustomResourceDefinition is terminating", requestInfo.Verb), http.StatusMethodNotAllowed)
```

Comment:
not via status? We have mixture, some things are returned as status, some are not.